src/modules/countLikes.js

```diff
@@ -0,0 +1,5 @@
+const countLikes = (countMovies, moviesLength) => {
+  countMovies.textContent += ` (${moviesLength})`;
+};
+
+export default countLikes;
```

Comment:
- [ ]  OPTIONAL
When I continue to click on the like button, it keeps the counter only keeps increasing. However, I think you can improve upon your _likes_ feature this way:
    - Like icon should toggle to indicate _like_ or _unlike_ when a particular user clicks on the like icon.
    - Total count of likes per user should be 1.

Can you try achieving this with the help of local storage or other suitable approaches?